survey/src/main/java/com.quickbirdstudios.surveykit/AnswerFormat.kt

```diff
@@ -117,6 +117,34 @@ sealed class AnswerFormat {
         }
     }
 
+    object LocationAnswerFormat : AnswerFormat() {
+
+        @Parcelize
+        data class Location(val latitude: Double, val longitude: Double) : Parcelable {
+            override fun toString(): String {
+                return "${this.latitude}:${this.longitude}"
+            }
+
+            override fun equals(other: Any?): Boolean {
```

Comment:
The `equals` and the `hashCode` are not necessary. The `toString` is only necessary if you want to print it that way specifically. Otherwise, data classes have their own `toString` implementation